rapids_pytest_benchmark/rapids_pytest_benchmark/plugin.py

```diff
@@ -472,6 +471,7 @@ def pytest_sessionfinish(session, exitstatus):
         commitTime = asvMetadata.get("commitTime", commitTime)
         commitRepo = asvMetadata.get("commitRepo", commitRepo)
         commitBranch = asvMetadata.get("commitBranch", commitBranch)
+        requirements = json.loads(asvMetadata.get("requirements", "{}"))
```

Comment:
I may have mentioned offline that this can be used as-is, but now I'm thinking that you can remove the `json.loads()` here since (I think) this was taken care of in `_parseSaveMetadata()` (ie. it'll already be a python dictionary).